tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php

```diff
@@ -1369,4 +1369,56 @@ public function provideWithConfigWithNoCandidateCases()
         yield ['z', '__sleep'];
         yield ['z', 'abc'];
     }
+
+    /**
+     * @param string $expected
+     * @param string $input
+     *
+     * @dataProvider provideFix80Cases
+     * @requires PHP 8.0
+     */
+    public function testFix80($expected, $input)
```

Comment:
@SpacePossum , can you elaborate on this PR?
it looks like not only fixing 5.6 job

maybe we can split it, having the 5.6 fix we merge first, and having 8.0 preparation merged separately ?